data-processing/.env

```diff
@@ -0,0 +1,5 @@
+POSTGRES_USER=postgres
```

Comment:
It's better if we don't duplicate files if not strictly necessary, and this file is already present in database/.env

I think you can grab database/.env instead? Let me know if I'm not correct!